### PR TITLE
Adjust BrowseRepoSettings

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

Minor layout issue.
Opened in designer too

### Before

image

### After

image

## Test methodology 

manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by RussKie — COMMENTED** [review]
I suggest rewording the text to something like "Show file history in the main window".
1. Use sentence case
2. Don't use internal type names. An average user shouldn't know how we name our types/dialogue.

**Comment by gerhardol** [comment]
> I suggest rewording the text to something like "Show file history in the main window".
> 
>     1. Use sentence case
> 
>     2. Don't use internal type names. An average user shouldn't know how we name our types/dialogue.

1. Changed in #9789 

2. FileHistory was used as that is what the user see in 3.5 when opening the file history.
Browse is used in the title for the page too.

Will change in #9789 (using suggestion), just keep the layout change here (there are enough conflicts in English.xlf already).

**Comment by RussKie** [comment]
👍
